README.md

```diff
@@ -45,7 +45,7 @@ Splice’s interlocking system of incentives and fees includes:<br>
 * Handles for amulet users, to make it easy to find counterparties on chain and exchange amulets with them. <br>
 
 ## Status
-The code that forms Splice was first implemented in June 2023 as part of a TestNet synchronizer operated by the members of an initial Super Validator Collective. At that time Splice was a set of applications with various names including Canton Coin and Canton Name Service. The TestNet synchronizer has operated continuously since that time, with regular tests and software upgrades, using the code proposed for Splice to implement its operations, governance and payment applications. In late December, Digital Asset and 46 separate financial institutions demonstrated 30 decentralized application deployments that synchronized roughly 350 financial transactions via the TestNet synchronizer. <br> <p>
+The code that forms Splice was first implemented in June 2023 as part of a TestNet synchronizer operated by the members of an initial  Validator Collective. At that time Splice was a set of applications with various names including Canton Coin and Canton Name Service. The TestNet synchronizer has operated continuously since that time, with regular tests and software upgrades, using the code proposed for Splice to implement its operations, governance and payment applications. In late December 2023, Digital Asset and 46 separate financial institutions demonstrated 30 decentralized application deployments that synchronized roughly 350 financial transactions via the TestNet synchronizer. <br> <p>
```

Comment:
```suggestion
The code that forms Splice was first implemented in June 2023 as part of a TestNet synchronizer operated by the members of an initial DSO. At that time Splice was a set of applications with various names including Canton Coin and Canton Name Service. The TestNet synchronizer has operated continuously since that time, with regular tests and software upgrades, using the code proposed for Splice to implement its operations, governance and payment applications. In late December 2023, Digital Asset and 46 separate financial institutions demonstrated 30 decentralized application deployments that synchronized roughly 350 financial transactions via the TestNet synchronizer. <br> <p>
```